From 0409b3c1929346d0eed4c5421b8f88c89226d81a Mon Sep 17 00:00:00 2001
From: Rakina Zata Amni <rakina@chromium.org>
Date: Mon, 15 Jun 2020 07:19:26 +0000
Subject: Add include to fix compilation in some cases

After crrev.com/c/2214588 compilation fails in some cases because of
usage of std::array, so this CL adds #include<array> to fix that.

Context: https://chromium-review.googlesource.com/c/chromium/src/+/2214588/8#message-ad38f798daa51f86fe4bc215657763fb21221add
Change-Id: I407faad7adb33ee37b4709e23172faf4dc1f5252
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2245437
Reviewed-by: Kinuko Yasuda <kinuko@chromium.org>
Commit-Queue: Rakina Zata Amni <rakina@chromium.org>
Cr-Commit-Position: refs/heads/master@{#778206}
---
 content/common/content_navigation_policy.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/content/common/content_navigation_policy.h b/content/common/content_navigation_policy.h
index b5d76ae4812a..e7d1aaedd765 100644
--- a/content/common/content_navigation_policy.h
+++ b/content/common/content_navigation_policy.h
@@ -7,6 +7,7 @@
 
 #include "content/common/content_export.h"
 
+#include <array>
 #include <string>
 
 namespace content {
-- 
2.20.1

